### ISSUE TITLE
Fake Error message when starting a session

### ISSUE THREAD

**Issue by RkDx** [ctxis/crackerjack#26]
Hello, when I start a session and the status is at running in the View Raw/Debug Information part, an error message is displayed on the main page as well as the session page itself instead of a running message.

![image](https://user-images.githubusercontent.com/61869865/211849976-77d042f4-044a-499d-807e-a5f4c98f46ee.png)

**Comment by RkDx** [comment]
The message changes if the session was successfully terminated by cracking the hash but it would be perfect if the user was warned of an ongoing session.

**Comment by sadreck** [comment]
You are right, the `error` state is more like a "fallback" if it it's not able to capture the correct one. I'll see what I can do to fix it - thank you for reporting it
